train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -46,4 +48,13 @@ public void setJoystickPosition(int joystickPosition) {
 		this.step = joystickPosition;		
 	}
 
+	@Override
+	public boolean setReferenceSpeed() {
```

Comment:
This method has already implemented in the TrainController class. Check the followSpeed function. ;)